pkg/controller/endpointslicemirroring/reconciler_test.go

```diff
@@ -880,6 +997,10 @@ func expectEndpointSlices(t *testing.T, num, maxEndpointsPerSubset int, endpoint
 		if serviceNameVal != endpoints.Name {
 			t.Errorf("Expected EndpointSlice to have %s label set to %s, got %s", discovery.LabelServiceName, endpoints.Name, serviceNameVal)
 		}
+
+		if epSlice.Annotations[corev1.LastAppliedConfigAnnotation] != "" {
```

Comment:
this doesn't fail if the annotation is set but doesn't return any value
https://play.golang.org/p/1wIZNGroJCT
```suggestion
		_,ok = epSlice.Annotations[corev1.LastAppliedConfigAnnotation] 
		if ok {
```